linera-client/src/client_context.rs

```diff
@@ -404,21 +404,15 @@ where
         }
     }
 
-    pub async fn assign_new_chain_to_key(
+    pub async fn chain_description(
```

Comment:
This function is exactly(?) like the one in `chain_listener.rs` 👀

Reply:
Oops!  I thought I'd deleted this.  Thanks :)